devito/ops/compiler.py

```diff
@@ -0,0 +1,16 @@
+from devito.parameters import configuration
+
+
+__all__ = ['CompilerOPS']
+
+
+class CompilerOPS(configuration['compiler'].__class__):
+    def __init__(self, *args, **kwargs):
+        super(CompilerOPS, self).__init__(*args, **kwargs)
+
+    def jit_compile(self, soname, code, hcode):
+        super(CompilerOPS, self).jit_compile(soname, code)
```

Comment:
This should not be called here

Reply:
My idea was using the already defined  ```jit_compile``` function of the Compiler class and only appending what I need the CompilerOPS to do (writing the .h file). What do you suggest?

Reply:
There's a few problems with this (and where you placed this):

- you need to call the ops translator
- you need to create the header file BEFORE compiling (therefore calling jit_compile from your superclass simply won't work) 
- what platform are we targeting? For example, CUDA would require multiple compilation steps (for host and device code)

My suggestion is to make this the base class of all OPS compilers where all we do is write the code to files and invoke the translator, then (in future PRs) subclass this compiler to create platform specific implementations. 

For some of these we might be able to reuse code from the `Compiler` class but for most we won't.

Reply:
Done! Now calling ops translator and generating code from it